lib/db/dao/message_dao.dart

```diff
@@ -282,13 +282,27 @@ class MessageDao extends DatabaseAccessor<MixinDatabase>
         [MiniTranscriptMessage(transcriptId: messageId)]);
   }
 
-  Future<void> deleteMessagesByConversationId(String conversationId) =>
-      db.transaction(() async {
-        await db.pinMessageDao.deleteByConversationId(conversationId);
-        await (delete(db.messages)
-              ..where((tbl) => tbl.conversationId.equals(conversationId)))
-            .go();
-      });
+  Future<void> deleteMessagesByConversationId(String conversationId) async {
+    await db.pinMessageDao.deleteByConversationId(conversationId);
+    while (true) {
```

Comment:
need add some log for this loop, to avoid CPU consume too much, but we can not found what happening.